qiskit/qobj/pulse_qobj.py

```diff
@@ -77,7 +77,7 @@ class PulseQobjInstruction:
     """A class representing a single instruction in an PulseQobj Experiment."""
 
     def __init__(self, name, t0, ch=None, conditional=None, val=None, phase=None,
-                 duration=None, qubits=None, memory_slot=None,
+                 frequency=None, duration=None, qubits=None, memory_slot=None,
```

Comment:
Can you move frequency to the end? The order of kwarg is part of the signature and while it's unlikely someone will be calling PulseQobjInstruction's constructor with just ordered args it's not worth potentially breaking someone over that.

Reply:
Resolved in cc781a1